library/db/src/main/kotlin/org/keynote/godtools/android/db/Contract.kt

```diff
@@ -179,6 +184,8 @@ object Contract : BaseContract() {
         internal const val SQL_V45_POPULATE_HIDDEN = "UPDATE $TABLE_NAME SET $COLUMN_HIDDEN = 0"
         internal const val SQL_V47_ALTER_SCREEN_SHARE_DISABLED =
             "ALTER TABLE $TABLE_NAME ADD COLUMN $SQL_COLUMN_SCREEN_SHARE_DISABLED"
+        internal const val SQL_V48_CREATE_SPOTLIGHT = "ALTER TABLE $TABLE_NAME ADD COLUMN $COLUMN_SPOTLIGHT"
```

Comment:
you should be using `SQL_COLUMN_SPOTLIGHT` here. you need to include the column type and not just the name of the column